specificatie/filter.yaml

```diff
@@ -26,7 +26,7 @@ components:
       description: |
         Gemeentenaam of een buitenlandse plaats of een plaatsbepaling, die aangeeft waar de persoon is geboren. **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
       type: string
-      pattern: ^[a-zA-Z0-9À-ž \,\.\-\'()]{1,40}$
+      pattern: ^[a-zA-Z0-9À-ž \,\.\-\'()]{1,80}$
       maxLength: 40
```

Comment:
we hebben al een pattern met maxLength 80, dan is daarnaast maxLength 40 niet handig (overbodig en strijdig)